wazo_phoned/plugins/gigaset/templates/gigaset_results.jinja

```diff
@@ -3,7 +3,9 @@
 <list response="get_list" type="pr" total="{{ total }}" first="{{ offset + 1 }}" last="{{ last }}">
 {%- for result in results %}
 <entry id="{{ result.number|e }}">
-<fn>{{ result.name|e }}</fn>
+{% set first_name, last_name = result.name.split(maxsplit=1) -%}
+<ln>{{ last_name|e }}</ln>
+<fn>{{ first_name|e }}</fn>
```

Comment:
I HATE this, the default display in dird should be modified to return a firstname, a lastname and the name field. All plugins should also be configured to return the firstname and lastname.

Then any ui could just display what it want :(